src/sentry/static/sentry/app/views/settings/account/apiTokens.jsx

```diff
@@ -122,4 +133,5 @@ class ApiTokens extends AsyncView {
   }
 }
 
-export default ApiTokens;
+export {ApiTokens};
```

Comment:
@leeandher Nit: move the export to the class so you have `export class ApiTokens`

Reply:
Done in latest 🔥